### DISCUSSION TITLE
montare disco su usb

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#398] [HUB: Home Assistant]
Ho Hassio su SSD collegato ad USB, ma non è di questo che chiedevo...
c'è un modo semplice, un add-on che permetta di vedere ed utilizzare un hard-disk collegato all'altra USB del mio raspberry?

**Comment by 50494554524F** [comment]
ho provato solo su macchina virtuale...

- dopo che ha si è avviato digita "login" ed entra nella console
- "lsblk -p" e vedi come si chiama lo hdd (sarà "sdc" visto che ne hai già un ssd)
- "fdisk /dev/sdc", cancella tutte le partizioni con il comando "d", poi "g" per creare la tabella partizioni, "n" per creare una partizione, poi "w" per salvare
- "mkfs -t ext4 /dev/sdc1" per creare il filesystem
- "mkdir /mnt/data/supervisor/share/pippo", ad esempio
- "mount /dev/sdc1 /mnt/data/supervisor/share/pippo"

al riavvio bisogna rifare mount... devo capire come renderlo persistente

**Reply by snakuzzo** [reply]
`/etc/fstab`

**Reply by 50494554524F** [reply]
ormai è chiario che devo (ri)studiare anche linux... non mi è mai piaciuto

**Reply by snakuzzo** [reply]
fattelo piacere...una volta capite le potenzialità butterai Windows

**Reply by 50494554524F** [reply]
sono due giorni che cerco... ma non c'è una man page dei comandi divisi per categoria?!? tipo comandi per disco, file etc...

**Reply by maxcanna** [reply]
Tu quando consulti una enciclopedia cerchi la voce della nazione che ti interessa o sfogli l'elenco delle nazioni cercando quella che ti può piacere conoscere meglio?

**Reply by 50494554524F** [reply]
e se ti interessa sapere le nazioni di un continente te che consulti?!?!?

**Reply by maxcanna** [reply]
Quello che sto cercando di dirti è che `man` non è un corso. È una reference. Se sai che il continente esiste e il suo nome cerchi i dettagli. Se tu non sai nemmeno quali continenti ci siano è meglio un corso di geografia. Tu stai cercando un qualcosa che ti dia una panoramica generale. Questo non è `man`. Chiudo qui perché siamo OT ☺

**Reply by 50494554524F** [reply]
allora riformulo... se voglio sapere la lista dei comandi per il disco (fdisk, mount etc...) dove vado a cercare?!

**Reply by Dregi56** [reply]
Allora, ora vi dico anche la mia, visto che nessun altro l'ha fatto, ve lo dico io: **Siete leggermente, non tanto eh(?) OFF-TOPIC** . E siccome qua dentro c'è qualcuno che si lamenta di un  messaggio in più sul forum, vi volevo dire che l'account è collegato ad una email e mi continuano ad arrivare messaggi sulla casella di posta. Per cui vi chiedo cortesemente di aprire un altro tread per dilettarvi. Vi ringrazio.

**Reply by snakuzzo** [reply]
@Dregi56 il tuo intervento è completamente fuori luogo. Se andiamo a riguardare tutti i topic "inutili" che hai aperto a cominciare da quando eravamo su Spectrum...si fa notte. Ti ho già ripetuto n volte che per le tue banalissime richieste per le quali basterebbe sapere dell'esistenza di Google o di un qualsiasi altro motore di ricerca, puoi provare a chiedere sulla chat Telegram.
Poi...tornando al tuo inutile intervento, se ti danno fastidio le notifiche è un problema tuo, non nostro. Se non ti sta bene, lamentati con github e comunque, per tua informazione, puoi tranquillamente impostare le notifiche a tuo piacimento.
Quindi...oltre a chiederti di non aprire topic ogni volta per qualsiasi cavolata, ti chiedo cortesemente anche di non intervenire a sproposito andando non "leggermente", ma "TOTALMENTE" off-topic.
E' l'ultimo avvertimento...stai veramente esagerando.

@50494554524F tornando alla tua richiesta sul "dove" andare a cercare, prova a dare un'occhiata [QUI](https://wiki.debian.org/it/fstab).
